### ISSUE TITLE
Add PRQL to SQL context menus to PRQL editor title

### ISSUE THREAD

**Issue by RandomFractals** [PRQL/prql-vscode#41]
This change would add direct access to the `prqlSqlOutputPanel.open` command to the open PRQL editor title menu and vscode  title bar context menu.

Required `package.json` changes to the extension `contributes` config:

```json
...
    "commands": [
      {
        "command": "prqlSqlOutputPanel.open",
        "title": "Open SQL Output Panel",
        "category": "PRQL",
        "icon": "$(database)"
      }
    ],
    "menus": {
      "editor/title": [
        {
          "command": "prqlSqlOutputPanel.open",
          "when": "resourceFilename =~ /.*(.prql)/",
          "group": "navigation"
        }
      ],
      "editor/title/context": [
        {
          "command": "prqlSqlOutputPanel.open",
          "when": "resourceFilename =~ /.*\\.(prql)/",
          "group": "navigation"
        }
      ]
    }
...
```

We can use the standard vscode `$(database}` icon to expose Open SQL Output Panel command in the top right corner of editor panel as depicted below:

![prql-to-sql-db-icon-context-menu](https://user-images.githubusercontent.com/656833/216652974-c1cbdd2d-4725-4136-a436-a1d148a094f1.png)

**Comment by max-sixty** [comment]
I think this would be great!

**Comment by RandomFractals** [comment]
Closing this as it was merged into main branch as part of PR #40.

We might consider changing editor title menu icon from `$(database)` to `$(open-preview)` and use the database icon later on for selection of database to connect to and run generated SQL.

I'll expand more on it in #43.
